### ISSUE TITLE
Add playwright tests to verify reactions and mentions from different users

### ISSUE THREAD

**Issue by Srutip04** [Srutip04/neeto-planner-playwright-tests-by-sruti-pingua#14]
### Steps:
1. **Visit Subdomain as Standard User:**
   - Log in to neetoPlanner as the standard user.
   - Go to the "Tasks" section on the sidebar and navigate to the Mentions and Reactions subsection.
   - Keep track of the notifications before starting the test.

2. **Visit Subdomain as Admin User:**
   - Log in to neetoPlanner as the admin user.
   - Click on the "Projects" section on the sidebar.
   - Create a new project and add the standard user to that project.
   - Create a new task within the project and assign it to the standard user.

3. **Add Comment Mentioning Standard User:**
   - After refreshing the page, open the details of the task.
   - Add a new comment to the task mentioning the standard user.

4. **Verify Mentions and Reactions as Standard User:**
   - Go to Tasks > Mentions and Reactions as the standard user.
   - Refresh the page and verify that the mention by the admin is visible.
   - Mark it as read and verify that the notification is no longer highlighted.
   - Click on the mention notification and ensure the details of the task open up with the comment highlighted.

5. **Add Comment Mentioning Admin:**
   - Add a new comment to the task mentioning the admin.

6. **Verify Mentions and Reactions as Admin User:**
   - Go to the mentions and reactions section as the admin user.
   - Refresh the page and verify a new notification is shown for the mention comment by the standard user.
   - Mark it as read and verify that the notification is no longer highlighted.
   - Click on the notification and ensure the pane with the correct task details opens, and the proper comment is highlighted.

7. **React to Comment with Emoji:**
   - React to the comment with the white checkmark emoji (✅).

8. **Verify Reaction as Standard User:**
   - Go to the mentions and reactions page as the standard user.
   - Refresh the page and verify the reaction is visible and highlighted as unread.
   - Mark it as read and verify that the notification is no longer highlighted.
   - Click on the notification and ensure the pane with the details of the task is displayed, and the comment with the emoji is highlighted.

### Expected Result:
- Notifications for mentions and reactions should be properly displayed and handled for both standard user and admin user interactions.
- Comments and reactions should be accurately reflected in the task details view.
